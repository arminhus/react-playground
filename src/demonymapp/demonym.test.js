import React from "react";
import ReactDom from "react-dom";
import renderer from "react-test-renderer";

import Demonym from "./demonym";

describe("Demonym component test", () => {
  test("should render demonym component", () => {
    const div = document.createElement("div");
    ReactDom.render(<Demonym />, div);
    ReactDom.unmountComponentAtNode(div);
  });

  test("should render with name and country", () => {
    const tree = renderer
      .create(<Demonym name="American" country="United States of America" />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
