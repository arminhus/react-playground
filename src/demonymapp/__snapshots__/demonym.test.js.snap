// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Demonym component test should render with name and country 1`] = `
<div
  className="demonym"
>
  A 
  American
   comes from 
  United States of America
  .
</div>
`;
